main/lang/english/trad4all.inc.php

```diff
@@ -142,6 +142,7 @@
 $SessionDurationEdit = "Edit session duration";
 $EditUserSessionDuration = "User's session duration edition";
 $SessionDurationXDaysLeft = "This session has a maximum duration. Only %s days to go.";
+$SessionDurationNotStarted = "This session has a maximum duration of %s.";
```

Comment:
New language variables have to go through another, more complex process (so that translators can participate). I've added the variable, but could you change your PR to:
- remove all main/lang/ changes
- use variable $SessionDurationNotStartedMaxX instead of $SessionDurationNotStarted (the "X" identifies that this string contains a %s token that must be replaced through a sprintf() call)

Thanks.